docs/release-notes/NuGet-6.2.md

```diff
@@ -0,0 +1,102 @@
+
+* Errors due to missing/failing sources are inconsistently shown in solution explorer vs the error list  - [#7245](https://github.com/NuGet/Home/issues/7245)
+
+* Arrow keys in NuGet PM UI Sources editing doesn't change order of persistence - [#8315](https://github.com/NuGet/Home/issues/8315)
+
+* PackageReference ungracefully handles duplicate Runtime Identifiers in csproj PackageReference  - [#9290](https://github.com/NuGet/Home/issues/9290)
+
+* RestoreIgnoreFailedSources=true still gives warnings - [#9765](https://github.com/NuGet/Home/issues/9765)
+
+* Introduce a warning for null/empty version range (new or reuse NU1604) - [#9767](https://github.com/NuGet/Home/issues/9767)
+
+* NuGet restore in VS should report progress to allow the project-system to better control design time build scheduling.  - [#9782](https://github.com/NuGet/Home/issues/9782)
+
+* NuGet again throwing exceptions "authors is required" "description is required", ignoring csproj/nuspec replacement tokens - [#9954](https://github.com/NuGet/Home/issues/9954)
+
+* [Regression]: Performance regression for cold restores in .NET 5.0.x - [#11031](https://github.com/NuGet/Home/issues/11031)
+
+* [Bug]: Package extraction sometimes fails with "file in use by another process" - [#11373](https://github.com/NuGet/Home/issues/11373)
+
+* Add progress reporting during package installation - [#11432](https://github.com/NuGet/Home/issues/11432)
+
+* [Bug]: Reduce string allocations in restore code path - [#11475](https://github.com/NuGet/Home/issues/11475)
+
+* [Bug]: Errors NU3028 and NU3037 when restoring NuGet packages on FreeBSD - [#11481](https://github.com/NuGet/Home/issues/11481)
+
+* [Responsiveness] RestoreOperationLogger blocking large number of thread pool threads trying to get access to the output window pane - [#11501](https://github.com/NuGet/Home/issues/11501)
+
+* [Bug]: Race Condition Creating Plugin Log Files - [#11517](https://github.com/NuGet/Home/issues/11517)
+
+* [Responsiveness] Package Management UI can consume large number of threads all searching the disk, it needs to run from long running thread - [#11570](https://github.com/NuGet/Home/issues/11570)
+
+* [Responsiveness] Package Management UI can consume large number of threads all searching the disk (up to 316 threads), use cancellation token at subroutines - [#11599](https://github.com/NuGet/Home/issues/11599)
+
+* [Bug]: NU1004 in Visual Studio, but not command line (lock files in locked mode) - [#11639](https://github.com/NuGet/Home/issues/11639)
+
+* [Bug]: new warning for package source mappings doesn't pass a value for the resource string placeholder - [#11709](https://github.com/NuGet/Home/issues/11709)
+
+**StillOpens:**
+
+* GeneratePathProperty properties should end with a trailing slash - [#8871](https://github.com/NuGet/Home/issues/8871)
```

Comment:
Still opens are meant to be deleted from the release notes/removed from the release.